pyrokinetics/examples/example_JETTO.py

```diff
@@ -14,7 +14,7 @@ def main(base_path: Union[os.PathLike, str] = "."):
     # Load up pyro object
     pyro = Pyro()
 
-    pyro.load_global_eq(eq_file=eq_file, eq_type="GEQDSK", psi_n_lcfs=1.0)
```

Comment:
The point of `psi_n_lcfs` was to artificially change where the actual LCFS is. The reason for this was that sometimes, when doing the contour fittings, if the actual value of `psi` was used to specify the LCFS, a connected contour wouldn't be found. So I gave the option of slightly shifting the value of `psi` for the LCFS.

Admittedly this is more of a issue with the contour fitting rather than the equilibrium but it was a simple workaround. I see that you can now you specify `psi_lcfs` directly. Can we add a multiplier to slightly change the value if needed. If you are loading in an equilibrium you likely won't know what `psi_lcfs` is but you could easily specify `psi_n_lcfs` which defines the `psi` used